cita-executor/src/main.rs

```diff
@@ -87,7 +87,7 @@ extern crate common_types as types;
 extern crate core_executor as core;
 #[macro_use]
 extern crate crossbeam_channel;
-extern crate db as cita_db;
+extern crate cita_database as cita_db;
```

Comment:
remove this `extern`?

Reply:
How to use `as` without `extern`?

Reply:
Try [renaming-dependencies-in-cargotoml](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)

Reply:
Try :
```rust
use cita_database as cita_db
```
?

Reply:
> Try [renaming-dependencies-in-cargotoml](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)

This works.